### PR TITLE
Look for Templates in the correct folder inside of themes

### PR DESCRIPTION
**Pull request by bordoni**
After a call with Victor I realized that we were looking into the wrong folder inside of the Users theme, so this Pull Request aims to fix that in a simple way.

### CODE REVIEW

File: src/Tribe/Template.php [hunk 1]
```diff
@@ -341,9 +369,17 @@ protected function get_template_public_namespace() {
 	 * @return string
 	 */
 	protected function get_template_public_path( $base ) {
+
 		// Craft the plugin Path
 		$path = array_merge( (array) $base, (array) $this->get_template_public_namespace() );
 
+		// Pick up if the folder needs to be aded to the public tempalte path
```
Comment (zbtirrell) [inline src/Tribe/Template.php]: minor comment typo

File: src/Tribe/Template.php [hunk 2]
```diff
@@ -331,6 +340,25 @@ protected function get_template_public_namespace() {
 		return apply_filters( 'tribe_template_public_namespace', $namespace, $this );
 	}
 
+	/**
+	 * Fetches which base folder we look for templates in the origin plugin.
+	 *
+	 * @since  TBD
+	 *
+	 * @return array
```
Comment (sc0ttkclark) [inline src/Tribe/Template.php]:
```suggestion
	 * @return array The base folders we look for templates in the origin plugin.
```

File: src/Tribe/Template.php [hunk 3]
```diff
@@ -331,6 +340,25 @@ protected function get_template_public_namespace() {
...
+	 */
+	public function get_template_origin_base_folder() {
+		/**
+		 * Allows filtering of the base path for templates
```
Comment (sc0ttkclark) [inline src/Tribe/Template.php]:
```suggestion
		 * Allows filtering of the base path for templates.
```

File: src/Tribe/Template.php [hunk 4]
```diff
@@ -331,6 +340,25 @@ protected function get_template_public_namespace() {
...
+		 *
+		 * @since TBD
+		 *
+		 * @param array  $namespace Which is the base folder we will look for files in the plugin.
+		 * @param self   $template  Current instance of the Tribe__Template
```
Comment (sc0ttkclark) [inline src/Tribe/Template.php]:
```suggestion
		 * @param self   $template  Current instance of the Tribe__Template.
```

File: src/Tribe/Template.php (hunk 1 above)
Comment (sc0ttkclark) [reply src/Tribe/Template.php]:
```suggestion
		// Pick up if the folder needs to be aded to the public template path.
```

**Pull request merged**